uiowa/uiowa
# Limiting tag and related content from displaying on all custom content types

Resolves #5909 

# How to test
With the event feature split enabled, add a tag to an event. See that the tag does not show on the rendered entity.
Add a tag to one of the node types expected to display (eg articles). See that the tag display still properly follows the site-wide settings
